documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -16,36 +16,77 @@ WebLogic Server supports two types of clustering configurations, configured and
+
 The operator provides several ways to initiate scaling of WebLogic clusters, including:
 
-* [On-demand, updating the Domain directly (using `kubectl`)](#on-demand-updating-the-domain-directly).
+* [kubectl CLI commands](#kubectl-cli-commands).
+* [On-demand, updating the Cluster or Domain directly (using `kubectl`)](#on-demand-updating-the-cluster-or-domain-directly).
 * [Calling the operator's REST scale API, for example, from `curl`](#calling-the-operators-rest-scale-api).
+* [Kubernetes Horizontal Pod Autoscalar (HPA)](#kubernetes-horizontal-pod-autoscalar-hpa).
 * [Using a WLDF policy rule and script action to call the operator's REST scale API](#using-a-wldf-policy-rule-and-script-action-to-call-the-operators-rest-scale-api).
 * [Using a Prometheus alert action to call the operator's REST scale API](#using-a-prometheus-alert-action-to-call-the-operators-rest-scale-api).
 
-#### On-demand, updating the Domain directly
-The easiest way to scale a WebLogic cluster in Kubernetes is to simply edit the `replicas` field of a Domain. This can be done by using `kubectl`. More specifically, you can modify the Domain directly by using the `kubectl edit` command.  For example:
+#### kubectl CLI commands
+The Kubernetes command-line tool, kubectl, can be used to manually scale WebLogic clusters with the following commands:
+
+* scale - Set a new size for a resource.
+* patch - Update field(s) of a resource using strategic merge patch.
+
+##### kubectl scale command
+An individual WebLogic cluster can be scaled directly using the `kubectl scale` command. For example, the following command sets .spec.replicas of the Cluster resource cluster-1 to 5:
 ```shell
-$ kubectl edit domain domain1 -n [namespace]
+$ kubectl scale --replicas=5 clusters/cluster-1
+  clusters "cluster-1" scaled
+ 
+$ kubectl get clusters cluster-1 -o jsonpath='{.spec.replicas}'
+  5
 ```
-Here we are editing a Domain named `domain1`. The `kubectl edit` command will open the Domain definition in an editor and allow you to modify the `replicas` value directly. Once committed, the operator will be notified of the change and will immediately attempt to scale the corresponding cluster by reconciling the number of running pods/Managed Server instances with the `replicas` value specification.
+
+##### kubectl patch command
+An individual WebLogic cluster can also be scaled directly using the `kubectl patch` command with your patch object inline. For example, the following command sets .spec.replicas of the Cluster resource `cluster-1` to 3:
+```shell
+$ kubectl patch cluster cluster-1 --type=merge -p '{"spec":{"replicas":3}}'
+  cluster.weblogic.oracle/cluster-1 patched
+ 
+$ kubectl get clusters cluster-1 -o jsonpath='{.spec.replicas}'
+  3
+```
+
+#### On-demand, updating the Cluster or Domain directly
+##### Updating the Cluster directly
```

Comment:
Per overall doc guide-lines, an intro sentence is required for the entire section. E.g. just after the '#### On-demand ...l' title and before the '##### Updating...' title. A brief sentence would be fine.